### ISSUE TITLE
Не приходит письмо с паролем пробую зарегестрироваться на другую почту

### ISSUE THREAD

**Issue by Artem1000I** [jenkins-infra/helpdesk#3343]
### Type(s) of account issue

I've been blocked by the anti-spam system

### Summary

Почта: artish111artish@gmail.com

### jenkins.io account username

artish111artish

### jenkins.io account email

artish111artish@gmail.com

### Which URL are you trying to login at?

[account.jenkins.io/antispam](https://accounts.jenkins.io/login)

**Comment by dduportal** [comment]
Hi @Artem1000I , this issue seems a duplicate of #3342. Can you explain why do you need 2 accounts please?

**Comment by Artem1000I** [comment]
I need one account. But the password does not come to the email I registered with. So I tried to register another account!

**Comment by Artem1000I** [comment]
> Hi @Artem1000I , this issue seems a duplicate of #3342. Can you explain why do you need 2 accounts please?

I need one account. But the password does not come to the email I registered with. So I tried to register another account!

**Comment by dduportal** [comment]
Thanks for the answer @Artem1000I . I've reset the password to the account `artish111artish`. An email was sent to this account: please check with your spams if you don't see anything on your inbox.
